sdk/search/search-documents/README.md

```diff
@@ -124,9 +128,9 @@ There are several types of operations that can be executed against the service:
 
 `SearchClient` provides methods for working with documents in an index. Its methods allow you to query, upload, update, and delete documents. It also has methods for building auto-completion and search suggestion experiences based on partial queries.
 
-### SearchServiceClient
+### SearchIndexClient & SearchIndexerClient
 
-`SearchServiceClient` provides methods for configuring and customizing an Azure Cognitive Search instance. The client currently has support for creating and managing search indexes and will later expand to support creating and managing other service entities such as indexers, synonym maps, cognitive skillsets, and data sources.
+`SearchIndexClient` and `SearchIndexerClient` provide methods for configuring and customizing an Azure Cognitive Search instance. The client currently has support for creating and managing search indexes and will later expand to support creating and managing other service entities such as indexers, synonym maps, cognitive skillsets, and data sources.
```

Comment:
this section doesn't really make sense, since after the first sentence it talks about a singular client singularly. Can we break this up into talking about each client separately?

Reply:
Done